pkg/pgmodel/ingestor/copier.go

```diff
@@ -174,7 +82,7 @@ func runInserter(conn pgxconn.PgxConn, in chan copyRequest) {
 	}
 }
 
-func inserterGetBatch(batch []copyRequest, in chan copyRequest) ([]copyRequest, bool) {
+func copierGetBatch(batch []copyRequest, in chan copyRequest) ([]copyRequest, bool) {
```

Comment:
Should this be `getBatch()` and in comment for this func, we write the copier stuff.

Reply:
I think current name is more descriptive